tests/unit/test_events.py

```diff
@@ -135,9 +152,10 @@ def distgit_commit(self):
     @pytest.fixture()
     def mock_config(self):
         service_config = ServiceConfig()
-        service_config.github_app_id = 123123
-        service_config.github_app_cert_path = None
-        service_config.github_token = "token"
+        service_config.services = {
+            GithubService(token="token"),
+            GitlabService(token="token"),
```

Comment:
Even with the config I am getting 

``` Authentication for url 'https://gitlab.com/testing-packit/hello-there' is missing in the config ```

Reply:
- Can you please rebase your PR? (We fixed an issue in CI so we can see the test results also here. The `tests_requre` still has a problem in CI.)
- Please, check that you have a fresh version of `OGR` since we fixed a bug there related to this.

Reply:
Marking as resolved.

The fixture was actually not used in the test... We already resolved it. (Any we can use the global, auto-used one introduced in #666 .)